ci/infra/testrunner/README.md

```diff
@@ -111,12 +114,29 @@ There are some arguments that are currently at the top level of the configuratio
 - nodeuser: the user name used to login into the platform nodes. Optional. 
 - ssh_key: specifies the location of the key used to access nodes. The default is to use the user's key located at `$HOME/.ssh/id_rsa`
 
+#### Packages
+The `packages` section configures the source of the packages to be installed in the nodes:
+
+* certificates: package with the certificates to be installed in the nodes in order to install other packages and images. For example, for installing SUSE self-signed packages:
+```
+packages:
+  certificates: "ca-certificates-suse"
```

Comment:
I don't understand why are we using certificates variable instead of something as generic as _package_ or _packages_. If I understood the code properly this `certificates` variable holds a package that will be provisioned by terraform at provisioning time. If so, a user could write something like `certificates: patterns-caasp-Node-1.15` and this would request  `patterns-caasp-Node-1.15` installation. Am I missing something here?

In fact, I'd love to have a way to include a custom set of packages in terraform, this way we could run the installation of several packages in parallel instead of installing them sequentially on each node (during node join) when deploying a cluster.

Reply:
Is a good point. Nothing prevents that. I can rename it ot something more general. Not to repeat `packages` inside`packages`, can we use `addons`? `extra`?

The same logic applies to the `maintenance` these are just registries. Should we follow the same logic?

Reply:
`extras` or `additional_packages` (`additional_pkgs` for shorter) sounds good to me :) 

About the `maintenance`, this is related to repositories right? Probably `additional_repos`? I am not that good naming stuff :sweat_smile:

In any case I believe that a more generic name is better if the logic is already generic. Then we could add specific examples to show the intended use case.

That's my view but I am not frontally against the current implementation.